bundles/org.openhab.binding.enturno/README.md

```diff
@@ -64,34 +64,34 @@ demo.items
 
 ```java
 // Stop place
-String      StopId                          "StopId"                        {channel="enturno:linestop:1:stopPlace#id"}               
-String      StopPlaceName                   "Stop Place [%s]"               {channel="enturno:linestop:1:stopPlace#name"}
-String      LineCode                        "Line [%s]"                     {channel="enturno:linestop:1:Direction01#lineCode"} 
-String      TransportMode                   "TransportMode [%s]"            {channel="enturno:linestop:1:stopPlace#transportMode"} 
+String      StopId                          "StopId"                        {channel="enturno:linestop:7e693fff:stopPlace#id"}
```

Comment:
Not sure why this was changed. It's just a random number that can be set by the user so any number should be ok. Also the thing above still uses 1.

Reply:
I can change it back to be 1 but if it's not a big thing then I will be grateful if I don't need to create new pull request just because of that.

Reply:
You need to do something because the thing example above still uses `1` and should have the same number.

Reply:
Ok, sorry, I see what you mean. I have updated id in thing configuration. I left this format however as it is similar to id created when you add thing in PaperUI.

Reply:
@Hilbrand Are you ok with review changes? I am planning to release widget for this binding today, so it would be great to have example in doc in sync with widget configuration.